ginga/canvas/types/astro.py

```diff
@@ -1274,6 +1275,11 @@ def draw(self, viewer):
             # and update all the text objects in self.objects
             update = True
 
+        cur_limits = viewer.get_limits()
+        if np.any(np.isclose(cur_limits, self._cur_limits)):
```

Comment:
In that case, this is more correct?
```suggestion
        if not np.all(np.isclose(cur_limits, self._cur_limits)):
```

Reply:
Right you are!